catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -318,6 +335,8 @@ public CatalogFrameworkImpl(BundleContext context, CatalogProvider catalogProvid
      *                                   response before calling any post-query plug-ins registered.
      * @param pool                       An ExecutorService used to manage threaded operations.
      * @param poller                     An {@link SourcePoller} used to poll source availability.
+     *
+     * @deprecated use {@link #CatalogFrameworkImpl(FrameworkProperties)}
```

Comment:
Is this class considered an external API? If not, can't we just delete those constructors?

If we need to keep them, I think we can safely remove the long and noisy `// NOTE` comments.